src/lunch.coffee

```diff
@@ -18,5 +18,6 @@ moment = require('moment-timezone')
 
 module.exports = (robot) ->
   robot.hear /lunch/i, (msg) ->
-    isAppropriateLunchTime = moment().tz("America/New_York").getHours() < 12
+    timezone = process.env.HUBOT_MOMENT_TIMEZONE || "Europe/London"
```

Comment:
Def a fan of having this be configurable.
2 small things:
1. Let's change the variable name to `HUBOT_LUNCHTIME_TIMEZONE` so it's specific to this plugin and won't conflict with other plugins.
2. Since the default was `America/New_York` to be begin with, let's keep the default as `America/New_York`

After those changes, I'd be more than glad to give a merge!